docs/tools/appium/setups-landing.md

```diff
@@ -35,6 +35,12 @@ Ruby is our most recent addition as for supported languages for Appium tests. As
     </ul></li>
 </ul>
 
+<h4 id="webdriverio">WebdriverIO</h4>
+WebdriverIO is a testing framework allowing you to write webtests using Javascript. Currently only the basic setup is supported.
```

Comment:
also expand by saying "Currently only the basic setup is supported. This means the result of your test won't be saved in your test report on TestObject." or so.